src/app/calendar/ParcelCalendar.tsx

```diff
@@ -0,0 +1,108 @@
+"use client";
+
+import supabase, { Schema } from "@/supabase";
+import Calendar, { CalendarEvent } from "@/components/Calendar/Calendar";
+import React, { useEffect, useState } from "react";
+import { styled } from "styled-components";
+
+interface ClientMap {
+    [primary_key: string]: Schema["clients"];
+}
+
+const Centerer = styled.div`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+`;
+
+const SizedAspectRatio = styled.div`
+    width: 100vmin;
+    margin: 2em;
+`;
```

Comment:
Not sure if this is intentional but this `div` doesn't maintain a constant aspect ratio

Reply:
Name changed to reflect the behaviour of the div